components/expander.jsx

```diff
@@ -1,12 +1,15 @@
 var React = require('react');
 
+var AnchorManagerMixin = require('../mixins/anchor-manager');
+
 var Expander = React.createClass({
```

Comment:
Hmm, so I have concerns with how helpful that actually is, since `children` is already a documented React property and the component won't actually crash or anything if `children` isn't defined...  We've actually had react warnings pop up in other components because `children` was declared to be `propTypes.array` instead of `propTypes.node`.

Aside from that, though, because we're using some stub context helper code to create the expander component in our tests, we can't easily add children to the expanders in our tests, and I don't want warnings logged to in our tests complaining that our expanders don't have children...